website/docs/pipelines/promoting-applications.mdx

```diff
@@ -185,13 +202,97 @@ type: Opaque
 ```
 
 :::note
-The Git provider token provided in the `token` field needs to be given permission to create pull requests in the pipeline's repository (defined in `.spec.promotion.strategy.pull-request.url`). The `hmac-key` field must match the key used for the Provider resource (.spec.secretRef), if specified in the leaf clusters.
+- The Git provider token provided in the `token` field needs to be given permission to create pull requests in the pipeline's repository (defined in `.spec.promotion.strategy.pull-request.url`).
+- The `hmac-key` field must match the key used for the Provider resource (.spec.secretRef), if specified in the leaf clusters.
 :::
 
+### Security Recommendations
+
 :::caution
-The Secret needs to reside in the same Namespace as the Pipeline resource on the management cluster.
+
+Ensure you adopt the recommendations in this section to reduce the risks associated with the secrets involved in pull requests.
 :::
 
+1. **Do not use user accounts**: use service or bot accounts. User context from the credentials would be used to do any git provider operation and for
```

Comment:
Bot accounts are "user accounts" in GitHub, perhaps this should say "create a user account specifically for making the changes"?

Reply:
reviewed, let me know if any better